gravity.sh

```diff
@@ -319,8 +319,12 @@ gravity_reload() {
 	#First escape forward slashes in the path:
 	adList=${adList//\//\\\/}
 	#Now replace the line in dnsmasq file
+<<<<<<< HEAD
```

Comment:
These lines need to be resolved, this is the merge conflict indicator. The lines between <<< and === are the same line as === to >>>> so you have to pick which set you think should stay as the valid, and delete the other lines. (And delete the <<< HEAD === and >>> lines.